stubs/Parsers/RulesParser.swift

```diff
@@ -31,18 +31,31 @@ enum JsonPathComponent {
 }
 
 enum Modification {
-    case add(dic: [String : Any])
+    case add(dic: [String: Any])
     case remove(keys: [String])
+    case transform(script: String)
 }
 
 struct ModificationRule {
     let path: [String]
     let modification: [Modification]
 
+    private let originalPath: [String]
+
+    var printablePath: String {
```

Comment:
This helps identify the exact rule that is having difficulties being transformed. I can go either way on this, tbh. I figured I would keep it here for now.

Reply:
Yes, this should be good for debugging